readme.md

```diff
@@ -75,7 +75,7 @@ rm -rf Monu.app Monu.zip # prevent duplicates in the final bundle
 npm run build
 
 > monu@1.0.4 build /Users/maxogden/src/js/monu
-> atom-shell-packager . Monu --ignore=node_modules/atom-shell
+> electron-packager . Monu --ignore=node_modules/electron-prebuild
```

Comment:
Oh, forgot to update that line with the new build args of `electron-packager`. But I think we can remove this code block. Doesn't make much sense to copy the whole `std output` in here. The important part are the git release infos and the success message.